### PR TITLE
#13 To connect travis-ci with slack notification

### PR DESCRIPTION
**Pull request by Wenfeng-GAO**
Hope all go well and world peace.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,3 +1,6 @@
 language: java
 jdk:
-  - oraclejdk7
+- oraclejdk7
```
Comment (mincong-h) [inline .travis.yml]: Did you change it with purpose?

Comment (Wenfeng-GAO) [inline .travis.yml]: Not really, file `.travis` was modified by this peace of code:
```travis encrypt "infiniot-team:<token>" --add notifications.slack```

Comment (mincong-h) [inline .travis.yml]: Ah OK. I think it's an auto-formatting and we can keep it. Where did this command launch? In your mac or in the server of Travis CI?

Comment (Wenfeng-GAO) [inline .travis.yml]: This peace of code was run in terminal of my Mac, after install `travis` cli from https://github.com/travis-ci/travis.rb#readme .
This line of code encrypted  slack team credential then modified the `.travis` file directly, so what I did is just commit and send pull request. :)

**Comment by mincong-h** [comment]
@Wenfeng-GAO, the next time, I'd like you to work with an issue in your fork and send the PR from your forked-repo. Avoid creating issue-branch in infiniot-web, which makes the official repo cleaner. You can rename the remote branches to `Wenfeng-GAO` and `infiniot`, so that there's no `origin`. It clarifies the different remotes.

**Comment by Wenfeng-GAO** [comment]
Yes, great suggestion!

**Pull request merged**